lib/player.rb

```diff
@@ -0,0 +1,11 @@
+require './lib/computer_choice'
+
+class Player
+  
+  attr_reader :name, :choice
+
+  def initialize(name = 'Computer', choice = ComputerChoice.new.choice)
```

Comment:
I like that you're using default params here as it makes using the computer choice really neat.  It might, however, be clearer as to what's going on if you explicitly passed these params in the controller.